ntbs-service/Pages/Search/Index.cshtml.cs

```diff
@@ -24,13 +27,22 @@ public class IndexModel : PageModel
         public string PreviousPageUrl;
         public string PreviousPageText;
 
+        [RegularExpression(@"[0-9]+", ErrorMessage = ValidationMessages.NumberFormat)]
+        [BindProperty(SupportsGet = true)]
+        public string IdFilter { get; set; }

```

Comment:
Might be worth putting all the search parameters into a searchParams class so we are just binding one property

Reply:
Very good idea, Do we start with this one? Or move it SearchParameters class when we add one more parameter?

Reply:
If we're about to start working on search together probably better to set it up here - I'll add it